### DISCUSSION TITLE
IMS not registered for operator Vivacom in Bulgaria

### DISCUSSION THREAD

**Discussion by iamnikv** [kyujin-cho/pixel-volte-patch#51] [Carrier-specific Experiences]
It turns out that the Pixel phones are not officially supported by Vivacom. Does this mean this project won't work in this case??
Vivacom supports VoLTE and VoWifi but for other brands. 

Is it a case of settings or it won't work if the operator does not support the device lineup?

**Comment by Galvaroz** [comment]
Is VoLTE and VoWiFi enabled by the operator?
As I understood this project just forcibly switches on these options on the phone and if the is a suitable config for you operator it will work.

**Comment by iamnikv** [comment]
I guess it's not, because sending sms commands for checking the status of VoLTE or trying to start it, result in saying that the service cannot be activated.

**Comment by eanavalentin** [comment]
So then why not have the SIM inserted into a supported phone, have the SMS sent for the services to activate and then go back to the Pixel and try the patch?
Once the SIM is provisioned by the operator there's a slight chance the services remain active.

**Comment by iamnikv** [comment]
Ok. I can try that!

**Comment by gezzior** [comment]
> So then why not have the SIM inserted into a supported phone, have the SMS sent for the services to activate and then go back to the Pixel and try the patch?
> Once the SIM is provisioned by the operator there's a slight chance the services remain active.

Hey! Not OP but I had the same problem and it helped me! Registered the card on a supported phone and now, after the patch, it works on my Pixel 7. Have a good day!

**Comment by iamnikv** [comment]
I enabled it on another phone but it still didn't work on my Pixel 7 Pro afterword.
